e2e_tests/test_suites/patch/test_setup.go

```diff
@@ -78,6 +78,13 @@ curl -X PUT --data "${new}" $uri -H "Metadata-Flavor: Google"
 	linuxLocalPrePatchScript = `
 echo 'curl -X PUT --data "1" http://metadata.google.internal/computeMetadata/v1/instance/guest-attributes/osconfig_tests/pre_step_ran -H "Metadata-Flavor: Google"' >> ./linux_local_pre_patch_script.sh
 chmod +x ./linux_local_pre_patch_script.sh
+`
+
+	setUpDowngradeScript = `
```

Comment:
Instead of making this a pre patch script just take these actions here. Your already setting this ins the startup script on line 115 _after_ osconfig gets installed so you should be safe to just pin the package directly and then you can remove the PreStep completly.

Reply:
Fixed